Model/Source/JobMethods.php

```diff
@@ -7,14 +7,18 @@ class JobMethods implements \Magento\Framework\Data\OptionSourceInterface
     private $methods = [
         'saveConfigurationToAlgolia' => 'Save Configuration',
         'moveIndexWithSetSettings' => 'Move Index',
-        'deleteObjects' => 'Object deletion',
-        'rebuildStoreCategoryIndex' => 'Category Reindex',
-        'rebuildCategoryIndex' => 'Category Reindex',
-        'rebuildStoreProductIndex' => 'Product Reindex',
-        'rebuildProductIndex' => 'Product Reindex',
-        'rebuildStoreAdditionalSectionsIndex' => 'Additional Section Reindex',
-        'rebuildStoreSuggestionIndex' => 'Suggestion Reindex',
-        'rebuildStorePageIndex' => 'Page Reindex',
+        'rebuildIndex' => 'Rebuild Full Index',
+        'rebuildIndexIds' => 'Rebuild Specific IDs',
```

Comment:
Instead of "Rebuild Specific IDs" (too technical sounding to me) what if we go with "Update index"?